views/components/ChartList.jsx

```diff
@@ -43,7 +44,11 @@ const ChartList = ({
             return (
               <TableRow key={chart._id}>
                 <TableCell>
-                  <Link color="textPrimary" href={routes.chart(chart._id)}>
+                  <Link
```

Comment:
it might be worth defining our own `Link` component that renders this `Link` with the `component={RouterLink}` prop so we don't need to re-write that for all of our links.